src/base/bittorrent/private/filterparserthread.h

```diff
@@ -37,6 +37,7 @@
 
 class QDataStream;
 class QStringList;
+template<typename T> class QVector;
```

Comment:
Although I've done it several times before, but now I doubt it. @evsh, is there any sense to forward declare templates?